### PR TITLE
Atc ascvd fhir data extraction

### PR DESCRIPTION
**Pull request by ibmatc**

### CODE REVIEW

File: ascvd-fhir-data-extraction/kubernetes.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  selector:
+    app: ascvd-fhir-data-extraction
+  type: NodePort
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: ascvd-fhir-data-extraction-ingress
+  annotations:
+    kubernetes.io/ingress.class: public-iks-k8s-nginx
```
Comment (ranum) [inline ascvd-fhir-data-extraction/kubernetes.yml:21]: maybe a general value for delivery-to be filled in later

File: ascvd-fhir-data-extraction/kubernetes.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+         # IBM: public-iks-k8s-nginx
+         # Azure: addon-http-application-routing
+         # AWS: nginx
+spec:
+  rules:
+    - host: atc-ascvd-fhir-data-extraction.integration-k8s-cluster-dcc48c44d831198cb8496b1ec68d7d12-0000.us-south.containers.appdomain.cloud
```
Comment (ranum) [inline ascvd-fhir-data-extraction/kubernetes.yml:27]: atc (you already saw this)

**Review by ranum — APPROVED** [review]
Looks ok to me.  A couple things you already know about.

File: ascvd-fhir-data-extraction/kubernetes.yml (hunk 1 above)
Comment (ibmatc) [reply ascvd-fhir-data-extraction/kubernetes.yml:21]: Fixed

File: ascvd-fhir-data-extraction/kubernetes.yml (hunk 2 above)
Comment (ibmatc) [reply ascvd-fhir-data-extraction/kubernetes.yml:27]: Fixed

**Pull request merged**